examples/asteroids/random_network_baseline.py

```diff
@@ -120,9 +120,11 @@
 i = 0
 try:
     while i < n:
-        pipeline.step()
+        result = pipeline.env_step()
+        pipeline.step(result)
 
-        if pipeline.done:
+        isDone = result[2]
+        if isDone:
             pipeline.reset_()
```

Comment:
It's usually bad practice to call pseudo-private methods directly, but this was already here 🤷‍♂

Reply:
Change `isDone` to `is_done` (PEP8). Also, AFAIK, methods pre-fixed with underscores pseudo-private, not post-fixed.

Reply:
Good catch, that was changed in another PR.

>Also, AFAIK, methods pre-fixed (not post-fixed) with underscores are pseudo-private.

I believe it's project dependent, unless there's some sort of PEP8 standard for it.
If it's not pseudo-private, the question becomes: why is it `reset_` and not simply `reset`?

Reply:
Good question. It's because I wanted to have a common function name for resetting `Network`s, `Nodes`, and `Connection`s, and many `Nodes` already had a `reset` attribute (referring to post spike reset voltage) at the time.

Reply:
Suppose originally `Nodes` had a typo and instead of `reset_`, it was called `resat`, and that you had already released this code and people were relying on `resat`.

I would advocate adding a new method called `reset`, keeping the `resat` around and marking it deprecated with a note to use `reset` for new code, and then naming all future reset methods as `reset`, _instead_ of doubling down on `resat` and naming all future reset methods as `resat`.

This is just my opinion, and I renamed `reset_` to `resat` to better illustrate the weirdness of doubling down on an inconsistent name, but I suppose it is what it is.

Reply:
I don't really see it as "doubling down", more of a lazy design choice when the code was just a grad school project. I don't mind changing the name of the method, but of course `resat` is no good. Maybe `reset_state_variables`?

Reply:
`reset_state_variables` might be good, depends how general/closed the method should be.

Some classes might want to do other things then just resetting the variables, but I suppose they could just have two different reset methods. 🤷‍♂